src/transformers/models/t5/tokenization_t5.py

```diff
@@ -24,6 +24,7 @@
 import sentencepiece as spm
 
 from ...tokenization_utils import PreTrainedTokenizer
+from ...tokenization_utils_base import TextInput
```

Comment:
Let's put that import in an `if TYPE_CHECKING` since it's just for a type annotation.